types/plist/index.d.ts

```diff
@@ -0,0 +1,30 @@
+// Type definitions for plist 3.0
+// Project: https://github.com/TooTallNate/node-plist#readme
+// Definitions by: Yusuke Higuchi <https://github.com/higuri>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+/// <reference types="node" />
+
+// plist
+export as namespace plist;
+
+// plist.parse()
+export function parse(xml: string): PlistValue;
+// plist.build()
+export function build(obj: PlistValue, opts?: PlistBuildOptions): string;
+
+// PlistValue
+export type PlistValue =
+  string | number | boolean |
+  Date | Buffer | PlistObject | PlistArray;
+export interface PlistObject {
+  [x: string]: PlistValue;
+}
+export interface PlistArray extends Array<PlistValue> { }
+
+// PlistBuildOptions
+//   The instance of this type is passed to 'xmlbuilder' module as it is.
+//   So we want to import this type from '@types/xmlbuilder',
+//   but the type is not declared there...
+export type PlistBuildOptions = object;
```

Comment:
Could you edit `@types/xmlbuilder` to declare it there? (I’ll take a look myself when I get off the bus)

Reply:
Thank you for your review. I'm trying it now (#29128).